README.md

```diff
@@ -1,2 +1,92 @@
+[wiki](https://github.com/enarx/enarx/wiki) in the
+[main Enarx repo](https://github.com/enarx/enarx/).
+
+In the future, you will be able to browse [the RFC Index](index.md) for
+a current listing of all RFCs and their statuses.
+
+There are 2 types of Enarx RFCs:
+
+* RFCs that describe individual features (in the [features](./features) folder)
+* RFCs that explain concepts underpinning many features
+(in the [concepts](./concepts) folder)
+
+## RFC Lifecycle
+
+RFCs go through a standard lifecycle.
+
+![lifecycle](lifecycle.png)
+
+#### PROPOSED
+To __propose__ an RFC, [use these instructions to raise a PR](
+contributing.md#how-to-propose-an-RFC) (Pull Request) against the repo.
+Proposed RFCs are considered a "work in progress", even after they are merged.
+In other words, they haven't been endorsed by the community yet, but they
+seem like reasonable ideas worth exploring.
+
+#### DEMONSTRATED
+__Demonstrated__ RFCs have one or more implementations available, listed in
+the "Implementations" section of the RFC document. As with the PROPOSED status,
+demonstrated RFCs haven't been endorsed by the community, but the ideas put
+forth have been more thoroughly explored through the implementation(s).
+The demonstrated status is an __optional__ step in the lifecycle.
+
+#### ACCEPTED
+To get an RFC __accepted__, [build consensus](contributing.md#how-to-get-an-RFC-accepted)
+for your RFC on [chat](https://gitter.im/enarx/community) and in community
+meetings. If your RFC is a feature that's code-related, it MUST have
+reasonable tests. An accepted RFC is incubating on a standards track;
+the community has decided to polish it and is exploring or pursuing implementation.
+
+#### ADOPTED
+To get an RFC __adopted__, [socialize and implement](contributing.md#how-to-get-an-rfc-adopted).
+An RFC gets this status once it has significant momentum: the mental model
+it advocates has begun to permeate our discourse and the Enarx code is a
+representation of that. In other words, adoption is acknowledgment of 
+ _de facto_ standard.
+
+To __refine__ an RFC, propose changes to it through additional PRs.
+Typically these changes are driven by experience that accumulates during or
+after adoption. Minor refinements that just improve clarity can happen inline
+with lightweight review. Status is still ADOPTED.
+
+#### RETIRED
+An RFC is __retired__ when it is withdrawn from community consideration by
+its authors, when implementation seems permanently stalled, or when
+significant refinements require a superseding document. If a retired RFC has been
+superseded, its `Superseded By` field should contain a link to the newer spec,
+and the newer spec's `Supersedes` field should contain a link to the older spec.
+Permalinks are not broken.
+
+### Changing an RFC Status
+
+See notes about this in [Contributing](contributing.md#changing-an-rfc-status).
+
+## About
+
+#### License
+
+This repository is licensed under an [Apache 2 License](LICENSE). 
```

Comment:
I'm having flashbacks to online math assignment submission portals, but I wonder if it should be Apache 2.0 instead of Apache 2... I'm not as familiar with Apache so I don't know if there already is or will be things like Apache 2.1, etc or if that's even important...

Reply:
I think I agree. Apache 2.0...

Reply:
Agreed, thanks. Nice catch.